python_modules/libraries/dagster-mysql/dagster_mysql_tests/docker-compose.yml

```diff
@@ -15,10 +15,10 @@ services:
     networks:
       - mysql
 
-  test-mysql-db-backcompat:
-    image: mysql:8.0.30
-    command: mysqld --default-authentication-plugin=mysql_native_password
-    container_name: test-mysql-db-backcompat
+  test-mysql-db-pinned:
+    image: mysql:8.0.21
+    command: mysqld --default-authentication-plugin=mysql_native_password --sql-require-primary-key=ON
```

Comment:
comments explaining these  - especially the backcompat doesnt work with flag bit